exercises/pangram/stack.yaml

```diff
@@ -0,0 +1 @@
+resolver: lts-8.6
```

Comment:
> either downgrade this one to lts-8.2, or upgrade the others

Let's save the upgrade for its own PR that affects all exercises like #500. I think this will be ready to merge.